tests/chainer_tests/functions_tests/rnn_tests/test_function_slstm.py

```diff
@@ -83,9 +83,6 @@ def setUp(self):
             self.check_forward_options = {'atol': 1e-3, 'rtol': 1e-2}
             self.check_backward_options = {'atol': 5e-3, 'rtol': 5e-2}
             self.check_double_backward_options = {'atol': 5e-3, 'rtol': 5e-2}
-        # TODO(dido1998) : Remove this skip
-        if self.grad_outputs[0] is False or self.grad_outputs[1] is False:
-            self.skip_double_backward_test = True
```

Comment:
This skip was required for SLSTM with ChainerX + fallback.
1. `None` is inserted in `self.inputs` https://github.com/chainer/chainer/blob/master/chainer/function_node.py#L466.
2. Non-`None` is https://github.com/chainer/chainer/blob/master/chainer/function.py#L216.